pkg/uroot/builder/bb/README.md

```diff
@@ -55,7 +55,7 @@ import (
 var name = flag.String("name", "", "Gimme name")
```

Comment:
for old time's sake could the default value of name be "Intruder alert"

That's what who on Unix would print when it could not figure out who you were.